packages/renderer/src/lib/container/ContainerColumnStatus.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+import PodIcon from '../images/PodIcon.svelte';
+import StatusIcon from '../images/StatusIcon.svelte';
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+</script>
+
+{#if 'type' in object && object.type === ContainerGroupInfoTypeUI.POD}
+  <StatusIcon icon="{PodIcon}" status="{object.status}" />
+{:else if 'type' in object && object.type === ContainerGroupInfoTypeUI.COMPOSE}
+  <StatusIcon icon="{PodIcon}" status="{object.status}" />
+{:else if 'state' in object}
+  <StatusIcon icon="{object.icon}" status="{object.state}" />
+{/if}
```

Comment:
```suggestion
{#if 'type' in object && typeof object.type === 'string'}
  <StatusIcon icon="{PodIcon}" status="{object.status}" />
{:else if 'state' in object && typeof object.type === 'string'}
  <StatusIcon icon="{object.icon}" status="{object.state}" />
{/if}
```

line 11 and 13 are the same, might be nice to simplify this

Reply:
I simplified in new commit, but just to `#if 'type' in object`. I assume you meant `typeof object.state` in the second case?  There are only two types here (ContainerInfoUI | ContainerGroupInfoUI) and type/state are each only on one, so I didn't really think checking the type of each property was necessary.

Reply:
Yeah having it everywhere would be redundant

Reply:
Added isContainerGroupInfoUI/isContainerInfoUI helper utils, and they use this logic.